src/components/sections/RerollCost.vue

```diff
@@ -29,7 +34,14 @@
             
             <tr>
               <th scope="row">Cost per 286</th>
-              <td scope="row" v-for="calculateCosts in getCalculateCostForRerollCount(286)" :key="calculateCosts">{{ calculateCosts }}</td>
+              <td scope="row" v-for="calculateCosts in getCalculateCostForRerollCount(pitySystemLimit)" :key="calculateCosts">{{ calculateCosts }}</td>
```

Comment:
@kabamgamer these keys aren't unique in a decent chunk of scenarios

Reply:
easiest one to repo would be 0 as values

Reply:
I've fixed this in my latest commit, slightly changed how we build the table to having a list of "chaosTiers" with title, mote amount and gold amount, using row prefix + title